ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -1054,7 +1054,10 @@ tests:
       TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&;ROSA&
       TEST_IMPORTANCE: Critical
     test:
-    - chain: openshift-e2e-test-hypershift-qe
+    - ref: cucushift-hypershift-extended-health-check
+    - ref: openshift-extended-test
+    - ref: openshift-extended-web-tests
```

Comment:
`ref: openshift-extended-web-tests` is currently unsupported for this type of configuration.